doc/check-options.md

```diff
@@ -0,0 +1,215 @@
+
+### no-autoplay-audio
+
+| Option            | Default | Description                                                                   |
+| ----------------- | :------ | :---------------------------------------------------------------------------- |
+| `allowedDuration` | `3`     | Maximum time an audio is allowed to autoplay before being marked as violation |
+
+### css-orientation-lock
+
+| Option            | Default | Description                                                                                                                            |
+| ----------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------- |
+| `degreeThreshold` | `3`     | The difference of degrees from 180 and 90 that are considered orientation lock (for example, 93° is orientation locked but 94° is not) |
+
+### meta-viewport-large
+
+| Option         | Default | Description                                                                                                |
+| -------------- | :------ | :--------------------------------------------------------------------------------------------------------- |
+| `scaleMinimum` | `5`     | The maximum CSS value of `scale-maximum` the check will apply to. Values above this number will be ignored |
+| `lowerBound`   | `2`     | The minimum CSS value of `scale-minimum` the check will apply to. Values below this number will be ignored |
+
+### meta-viewport
+
+| Option         | Default | Description                                                                                                |
+| -------------- | :------ | :--------------------------------------------------------------------------------------------------------- |
+| `scaleMinimum` | `2`     | The maximum CSS value of `scale-maximum` the check will apply to. Values above this number will be ignored |
+
+### header-present
+
+| Option     | Default                                                                                                          | Description                                        |
+| ---------- | :--------------------------------------------------------------------------------------------------------------- | :------------------------------------------------- |
+| `selector` | `h1:not([role]), h2:not([role]), h3:not([role]), h4:not([role]), h5:not([role]), h6:not([role]), [role=heading]` | Selector used to determine if a page has a heading |
+
+### landmark
+
+| Option     | Default             | Description                                                |
+| ---------- | :------------------ | :--------------------------------------------------------- |
+| `selector` | `main, [role=main]` | Selector used to determine if a page has a landmark region |
+
+### p-as-heading
+
+| Option | Default | Description |
+| ------ | :------ | :---------- |
+
```

Comment:
| `margins` | CSS properties {	"weight": 150, "italic": true },	{ "weight": 150,	"size": 1.15	}, { "italic": true, "size": 1.15	}, { "size": 1.4 } | Common CSS values used to display `p` elements as `h1`-`h6` elements determining if a `p` element is being improperly repurposed |